third_party/nvfuser/python/__init__.py

```diff
@@ -0,0 +1 @@
+from ._C import *
```

Comment:
I think you will need to remove `./nvfuser/__init__.py`, since this file is supposed to overwrite that.

Reply:
I didn't add this to the PR, I don't think.

Reply:
I understand, I did delete the file in the PR.

Reply:
Ha, now I see the file is `moved`. sorry I missed it the first time.